plugins/source/snyk/resources/services/reporting/issues.go

```diff
@@ -44,7 +44,7 @@ func fetchIssues(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource
 
 	req := snyk.ListReportingIssuesRequest{
 		Page:    1,
-		PerPage: 1000,
+		PerPage: 1,
```

Comment:
why only 1?

Reply:
Ah, accident; I changed this during testing. Good catch thanks!